task/09-functions-n-closures-tasks.js

```diff
@@ -171,7 +214,10 @@ function partialUsingArguments(fn) {
  *   getId10() => 11
  */
 function getIdGeneratorFunction(startFrom) {
-    throw new Error('Not implemented');
+    return () => 
+    {
```

Comment:
Curly brackets are redundant in this arrow function.

Reply:
Fixed